docs/src/references/c8y-configuration-management.md

```diff
@@ -72,20 +73,26 @@ The `c8y_configuration_plugin` configuration is stored by default under `/etc/te
 This [TOML](https://toml.io/en/) file defines the list of files to be managed from the cloud tenant.
 Each configuration file is defined by a record with:
 * The full `path` to the file.
+* An optional configuration `type`. If not provided, the `path` is used a `type`.
+* Optional unix file ownership: `user`, `group` and octal `mode`.  
+  These are only used when a configuration file pushed from the cloud doesn't exist on the device.
+  When a configuration file is already present on the device, this plugin never changes file ownership,
+  ignoring these parameters.
 
 ```shell
 $ cat /etc/tedge/c8y/c8y-configuration-plugin.toml
 files = [
-    { path = '/etc/tedge/tedge.toml' },
+    { path = '/etc/tedge/tedge.toml', type = 'tedge.toml' },
     { path = '/etc/tedge/mosquitto-conf/c8y-bridge.conf' },
     { path = '/etc/tedge/mosquitto-conf/tedge-mosquitto.conf' },
-    { path = '/etc/mosquitto/mosquitto.conf' }
+    { path = '/etc/mosquitto/mosquitto.conf', type = 'mosquitto'}
```

Comment:
I would add `user`, `group`, and `mode` example.
```suggestion
    { path = '/etc/mosquitto/mosquitto.conf', type = 'mosquitto', user = 'mosquitto', group = 'mosquitto', mode = 0o644 }
```

Reply:
Added